jest.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  testEnvironment: "jsdom",
+  transform: {
+    "^.+\\.tsx?$": "ts-jest",
+  },
+  globals: {
+    "ts-jest": {
+      tsconfig: "tsconfig.json",
+      isolatedModules: true,
+    },
+  },
+  // ...
```

Comment:
?

Reply:
@matt-greenfield-15  the commented out .. ?